src/custom/utils/analytics/settingsEvents.ts

```diff
@@ -0,0 +1,45 @@
+import { Category, _reportEvent } from './index'
+import { debounce } from 'utils/misc'
+
+export function toggleExpertModeAnalytics(enable: boolean) {
+  _reportEvent({
+    category: Category.EXPERT_MODE,
+    action: `${enable ? 'Enable' : 'Disable'} Expert Mode`,
+  })
+}
+
+export function showExpertModeConfirmationAnalytics() {
+  _reportEvent({
+    category: Category.EXPERT_MODE,
+    action: 'Show Confirmation',
+  })
+}
+
+export function toggleRecepientAddressAnalytics(isDisabling: boolean) {
```

Comment:
should we be coherent? if we add a flag, in line 4 you have `enable` and here is `isDisabling` 
We should probably go everywhere with `enable`?